package.json

```diff
@@ -70,20 +70,27 @@
   "homepage": "https://github.com/code-chunks/angular2-logger#readme",
   "dependencies": {},
   "devDependencies": {
-    "angular2": "^2.0.0-beta.17",
+    "@angular/common": "^2.0.0-rc.0",
+    "@angular/compiler": "^2.0.0-rc.0",
+    "@angular/core": "^2.0.0-rc.0",
+    "@angular/platform-browser": "^2.0.0-rc.0",
+    "@angular/platform-browser-dynamic": "^2.0.0-rc.0",
     "es6-promise": "^3.0.2",
     "es6-shim": "^0.35.0",
+    "zone.js": "^0.6.12",
     "reflect-metadata": "0.1.2",
     "rimraf": "^2.5.2",
-    "rxjs": "5.0.0-beta.6",
```

Comment:
I saw travis failed because of missing rxjs dependency, but it works fine on my environement when I do a clean npm i (removed node_modules to be sure it's installing a clean one). I expected npm to automatically resolve dependencies but it seems like travis's npm doesn't. I am using npm 2.15.1.